llvm/test/lab2/shmelev_ivan/check.ll

```diff
@@ -0,0 +1,172 @@
+; RUN: opt -load-pass-plugin %llvmshlibdir/ShmelevInstrumnetF%pluginext -passes=instrument-functions -S %s | FileCheck %s
```

Comment:
Please, check your solution on this IR: https://godbolt.org/z/Kaazc4Gao

and add similar test

Reply:
@aobolensk, done.

Reply:
@aobolensk, done.